.clippy.toml

```diff
@@ -0,0 +1 @@
+large-error-threshold = 192
```

Comment:
I'm not super happy about this, but I'm not sure we can easily avoid it either? Wrapping everything in `Arc<T>` is kinda lousy, and we don't have a deep error stack – each of these will be allocated once per request, which is IMHO reasonable?

Reply:
The `DeviceIsOfflineError` is there because the app might want to do something with the type information.

We could reasonably replace it with a `type` field on `GenericError`, and that could help. 

I'm just not sure how much we want to de-specialize our error types? I worry we'll just end up overloading `GenericError` with a ton of extra data and end up right back where we started.